build.sh

```diff
@@ -184,8 +187,8 @@ system_info:
   package_mirrors:
   - arches: [default]
     failsafe:
-      primary: http://deb.debian.org/debian
-      security: http://security.debian.org/
+      primary: http://raspbian.raspberrypi.org/raspbian/
```

Comment:
Hmm, I had some problems with `http://raspbian.raspberrypi.org/raspbian/` when enabling `arm64` builds and switched to `http://deb.debian.org/debian` which worked fine for both `arm32` and `arm64`, see https://github.com/timebertt/pi-cloud-init/commit/56aff01436f74a3a883c26367c359043e64de3c1#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823R184